.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
such a files, not related to a project, should be ignored in `.git/info/exclude` or in Your global gitignore `git config --global core.excludesFile 'Path_to_gitignore'`